extension/src/features/import_from_seed.rs

```diff
@@ -113,7 +123,18 @@ pub fn import_from_seed() -> Html {
         })
     };
 
-    seed_value.resize(24, String::default());
+    let seed = Rc::new(seed);
+    fn on_change_seed(
+        seed: Rc<UseStateHandle<Vec<String>>>,
+        index: usize,
+    ) -> Callback<Result<String>> {
+        Callback::from(move |value: Result<String>| {
+            let mut seed_value = (**seed).to_owned();
```

Comment:
mmm good point. I think I did this because I don't want to mutate the original vector. But I could make only one copy outside and share that as mutable

Reply:
Cool! if you think it doesnt makes sense, just leave it as it is now... since it is an extension i don't think this might cause any problem tbh

Reply:
Well, after a lot of tries, it was simpler than I expected XD